pcsx2/gui/GlobalCommands.cpp

```diff
@@ -86,15 +86,15 @@ namespace Implementations
 		{
 			g_Conf->EmuOptions.GS.FrameLimitEnable = true;
 			g_LimiterMode = Limit_Turbo;
-			g_Conf->EmuOptions.GS.LimitScalar = g_Conf->Framerate.TurboScalar;
+			gsUpdateFrequency(g_Conf->EmuOptions);
```

Comment:
It will conflict with my VSYNC PR. But would it be possible to move this new line outside of all the if/else branches.